basics/answers.md

```diff
@@ -156,7 +156,16 @@ isn't exactly like mine but it works just fine it's ok, and you can
     print(message, "!!!")
     print(message, "!!!")
     ```
-
+3. Here, we create  a new variable `palindrome_check`.We use this variable to store the  reversed  string(of`palindrome_input`). Then we check if the string that was given(`palindrome_input`)
+   and the reversed sting(`palindrome_check`) is the same using a simple `if loop`
+   ```python
+   palindrome_input=input("Type the number to check:") 
+   palindrome_check=palindrome_input[::-1] #Reverses the string
+   if palindrome_input==palindrome_check:
```

Comment:
We don't need an unnecessary variable. We just want to check if the string itself is same as after reversing.

```suggestion
   if palindrome_input == palindrome_input[::-1]:
```

The explanation should be updated too.